crates/conformance-tests/src/lib.rs

```diff
@@ -62,7 +62,7 @@ pub fn tests(tests_dir: &Path) -> anyhow::Result<impl Iterator<Item = Test>> {
             let config = r#try!(json5::from_str::<config::TestConfig>(&config)
                 .context("test config could not be parsed"));
 
-            let component_name = format!("{name}.wasm");
+            let component_name = "component.wasm";
```

Comment:
Why are we statically naming the file now?

Reply:
We were previously tying the name of the component to the name of the test which is no longer always the case. For example, the `outbound-wasi-http-v0.2.0-no-perms` test uses the `outbound-wasi-http-v0.2.0` component since the only difference between the `no-perms` test and the regular `outbound-wasi-http-v0.2.0` test is that the former's spin.toml doesn't grant the app the correct permission.